app/menu.js

```diff
@@ -10,7 +10,7 @@ exports.buildAppMenu = modulesPromise => {
     .reduce((menuItems, module) => {
       if (module.link) {
         return menuItems.concat({
-          label: module.name,
+          label: module.name + ` | ${module.link.split(':')[2].replace('/', '')}`,
```

Comment:
On modern js it is better to avoid using string concatenation and use the string template. In this case there is mix of the template and concatenation togther